### PR TITLE
docs: Add documentation for environment in CLI

### LINKED ISSUE #457: DOCS: Add docs for `environment` in CLI
## Description

Add documentation for the `environment` command in our CLI.

## Solution

- Create a file named `environment.md` in [here](https://github.com/keyshade-xyz/keyshade/tree/develop/docs/cli)
- Check the [commands](https://github.com/keyshade-xyz/keyshade/blob/develop/apps/cli/src/commands/environment.command.ts) and put up the docs accordingly
- You can take reference from [here](https://docs.keyshade.xyz/cli/profile)

### PR DESCRIPTION
**Pull request by Allan2000-Git**
## Description

This PR adds documentation for the new environment feature in the CLI, providing users with clear instructions on how to utilize this functionality effectively.

Fixes #457 

## Dependencies

No new dependencies introduced.

## Future Improvements

Consider adding examples for common use cases of the scan feature in future documentation updates.

## Mentions

@rajdip-b 

## Screenshots of relevant screens

N/A

## Developer's checklist

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-check on my work

**If changes are made in the code:**

- [ ] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [ ] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR
- [ ] There are no UI/UX issues

### Documentation Update

- [x] This PR requires an update to the documentation at [docs.keyshade.xyz](https://docs.keyshade.xyz)
- [x] I have made the necessary updates to the documentation, or no documentation changes are required.

___

### **PR Type**
Documentation

___

### **Description**
- Added a new documentation file for the `environment` command in the Keyshade CLI.
- Documented subcommands including `create`, `delete`, `get`, `list`, and `update`.
- Provided detailed usage instructions, arguments, and options for each subcommand.
- Enhanced user understanding of managing environments within Keyshade.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>environment.md</strong><dd><code>Add comprehensive documentation for `environment` CLI command</code></dd></summary>
<hr>

docs/cli/environment.md

<li>Added documentation for the <code>environment</code> command in Keyshade CLI.<br> <li> Detailed subcommands: <code>create</code>, <code>delete</code>, <code>get</code>, <code>list</code>, <code>update</code>.<br> <li> Provided usage examples and options for each subcommand.<br> <li> Included arguments and options descriptions for better understanding.<br>

</details>

  </td>
  <td><a href="https://github.com/keyshade-xyz/keyshade/pull/462/files#diff-aa02278b1b505668b45f80ea1a787d4ba147242b62e81c91353923f14585b4ba">+126/-0</a>&nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by rajdip-b** [comment]
Needs a few changes, I'll mark them late tonight.

File: docs/cli/environment.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+description: The `environment` command
+---
+
+# `environment`
+
+The `environment` command is used to manage environments in Keyshade. An environment is a context in which secrets are stored and accessed. It has a set of subcommands:
```
Comment (rajdip-b) [inline docs/cli/environment.md:7]: ~context in which secrets are stored and accessed~
context in which secrets and variables are stored and accessed

**Review by rajdip-b — CHANGES_REQUESTED** [review]

File: docs/cli/environment.md (hunk 1 above)
Comment (Allan2000-Git) [reply docs/cli/environment.md:7]: will make the changes asap.

**Comment by Allan2000-Git** [comment]
@rajdip-b I have made the changes.

**Review by rajdip-b — APPROVED** [review]

**Pull request merged**